packages/jest-haste-map/src/getMockName.js

```diff
@@ -11,11 +11,11 @@
 
 const path = require('path');
 
-const mocksPattern = '__mocks__';
+const MOCKS_PATTERN = path.sep + '__mocks__' + path.sep;
```

Comment:
Maybe we could hoist `MOCKS_PATTERN` it to `jest-util`? It appears in 2 packages and 3 different places.